share/cmake/modules/FindFastFloat.cmake

```diff
@@ -0,0 +1,129 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.
+#
+# Locate or install the fast_float library
+#
+# Variables defined by this module:
+#   fast_float_FOUND - If FALSE, do not try to include fast_float.h
+#   fast_float_INCLUDE_DIR - Where to find fast_float.h
+#   fast_float_VERSION - The version of the library (if available)
+#
+# Targets defined by this module:
+#   fast_float::fast_float - IMPORTED target, if found
+#
+# The library is include-only, there is no associated binary.
+#
+# If fast_float is not installed in a standard path, you can use the 
+# fast_float_ROOT variable to tell CMake where to find it. If it is not found 
+# and OCIO_INSTALL_EXT_PACKAGES is set to MISSING or ALL, fast_float will be 
+# downloaded, built, and statically-linked into libOpenColorIO at build time.
+#
+
+###############################################################################
+### Try to find package ###
+
+if(NOT OCIO_INSTALL_EXT_PACKAGES STREQUAL ALL)
+    set(_FastFloat_REQUIRED_VARS FastFloat_INCLUDE_DIR)
+
+    if(NOT DEFINED FastFloat_ROOT)
+        # Search for FastFloatConfig.cmake
+        # Do notice that the CMake Config module is "FastFloat"
+        # while the library is "fast_float"
+        # (capital letters replaced by lowercase and underscore)
+        find_package(FastFloat ${FastFloat_FIND_VERSION} CONFIG QUIET)
+    endif()
+
+    # There is no pkg-config support on fast_float.
+
+    # Override REQUIRED if package can be installed
+    if(OCIO_INSTALL_EXT_PACKAGES STREQUAL MISSING)
+        set(FastFloat_FIND_REQUIRED FALSE)
+    endif()
+
+    include(FindPackageHandleStandardArgs)
+    find_package_handle_standard_args(FastFloat
+        REQUIRED_VARS 
+            ${_FastFloat_REQUIRED_VARS}
+        VERSION_VAR
+            FastFloat_VERSION
+    )
+endif()
+
+###############################################################################
+### Create target
+
+if (NOT TARGET fast_float::fast_float)
+    add_library(fast_float::fast_float INTERFACE IMPORTED GLOBAL)
+    set(_FastFloat_TARGET_CREATE TRUE)
+endif()
+
+###############################################################################
+### Install package from source ###
+
+if(NOT FastFloat_FOUND)
+    include(ExternalProject)
+    include(GNUInstallDirs)
+
+    set(_EXT_DIST_ROOT "${CMAKE_BINARY_DIR}/ext/dist")
+    set(_EXT_BUILD_ROOT "${CMAKE_BINARY_DIR}/ext/build")
+
+    # Set find_package standard args
+    set(FastFloat_FOUND TRUE)
+    set(FastFloat_VERSION ${FastFloat_FIND_VERSION})
+    set(FastFloat_INCLUDE_DIR "${_EXT_DIST_ROOT}/${CMAKE_INSTALL_INCLUDEDIR}")
+
+    # This library is include-only. No need to add further flags.
+
+    if(_FastFloat_TARGET_CREATE)
```

Comment:
As the code now looks for the header only, the line 78 to line 99 seems now useless.